app/overrides/spree/admin/variants/_form/add_stock_management.html.haml.deface

```diff
@@ -0,0 +1,11 @@
+/ insert_bottom "[data-hook='admin_variant_form_fields']"
+
+- if Spree::Config[:track_inventory_levels]
+  .field.checkbox{ "data-hook" => "on_hand" }
```

Comment:
oh, that's just copied from #3613. it will be removed from this PR once #3613 is merged. It's  feedback for https://github.com/openfoodfoundation/openfoodnetwork/pull/3613/files#diff-857364a580104152130389111473282bR6
I'll address there.